### PR TITLE
Introduce Jacoco to check coverage

### PR DESCRIPTION
**Pull request by KengoTODA**
Introduce jacoco plugin and coveralls plugin, to check test coverage.
To enable coveralls, we need to link this repository to coveralls, but it seems that my account cannot link.

Example: 
* https://coveralls.io/builds/9621402
* https://coveralls.io/github/KengoTODA/spotbugs

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,3 +26,8 @@ wrapper {
   gradleVersion = '3.2.1'
   distributionType = Wrapper.DistributionType.ALL
 }
+
+coveralls {
```
Comment (jsotuyod) [inline build.gradle]: why is this at the top level `build.gradle` when it's configured specifically for only one submodule?

Comment (KengoTODA) [inline build.gradle]: I met build error when I put this to `findbugs/build.gradle` and couldn't understand its meaning. I will retry later.

One more: I forgot checking coverage in eclipse plugin project. I'll try to check coverage and [merge the result from multiple projects](https://github.com/kt3k/coveralls-gradle-plugin#use-with-mutli-project-build).

Comment (KengoTODA) [inline build.gradle]: I found no test in `eclipsePlugin`, I will not add it to target of coverage.

**Comment by KengoTODA** [comment]
@jsotuyod @henrik242 please check this pull request later, thanks in advance!

**Comment by KengoTODA** [comment]
@jsotuyod Could you have a try to link this repository to coveralls? I guess following link should work.

https://coveralls.io/repos/new?name=spotbugs&service=github

**Comment by jsotuyod** [comment]
@KengoTODA there you go! 

https://coveralls.io/github/spotbugs/spotbugs

**Comment by KengoTODA** [comment]
@jsotuyod Thanks, now it works!

**Pull request merged**